alfasim_sdk_api/api.h

```diff
@@ -339,8 +339,8 @@ DLL_EXPORT int get_state_variable_array(
 /*!
     Gets the current contents of a given secondary variable (For an array data pointer).
     A secondary variable is any variable calculated in the solver iterative procedure.
-    Note that not all variables are available at any time. If a given variable is not available,
-    a #NOT_AVAILABLE_DATA error is returned.
+    Note that not all variables are available at any time. If a given variable is not available
+    in one of the chosen scopes, a #NOT_AVAILABLE_DATA error is returned.
 
     List of `variable_name` values:
     - `"rho"`: Density [kg/m3]
```

Comment:
@arthursoprana please take a look at this variable names list and see if there are other variables that you think would be relevant to add here.